packages/meld.rb

```diff
@@ -0,0 +1,39 @@
+require 'package'
+
+class Meld < Package
+  description 'Meld is a visual diff and merge tool targeted at developers.'
+  homepage 'https://meldmerge.org/'
+  version '3.22.0'
+  license 'GPL-2'
+  compatibility 'all'
+  source_url 'https://gitlab.gnome.org/GNOME/meld/-/archive/3.22.0/meld-3.22.0.tar.gz'
```

Comment:
Use git?

Reply:
It doesn't matter! As long as we're getting the source from gitlab/github, either is functionally equivalent.